js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js

```diff
@@ -1123,12 +1126,27 @@ var DraggingHeaderViewController = modules.ViewController.inherit({
         return this._columnsController.allowMoveColumn(parameters.sourceColumnIndex, parameters.targetColumnIndex, parameters.sourceLocation, parameters.targetLocation);
     },
 
+    drag: function(parameters) {
+        var sourceIndex = parameters.sourceIndex,
+            sourceLocation = parameters.sourceLocation,
+            sourceColumnElement = parameters.sourceColumnElement,
+            headersView = this._columnHeadersView,
+            rowsView = this._rowsView;
+
+        if(sourceColumnElement) {
+            sourceColumnElement.css({ opacity: 0.5 });
```

Comment:
I think it would be better to extract the value to a constant

Reply:
I am fixed this